recipes/oxli/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "oxli" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/oxli-{{ version }}.tar.gz
+  sha256: fc695d891d0443e2af198c9808759f1984fcbf78d55f4be3d7b563474dd4e539
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win or py<310]
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - crossenv                            # [build_platform != target_platform]
+    - maturin >=1.1.0,<2                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }} >=1.65.0
+    - posix                               # [win]
+    - make                                # [unix]
+    - cargo-bundle-licenses
+    - libclang                            # [unix]
+    - clangdev                            # [unix]
+    - xz
+  host:
+    - python
+    - maturin >=1.4,<2.0
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - oxli
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dib-lab/oxli
+  summary: k-mer tools
```

Comment:
We usually ask for shorter summary but I confess this one made me almost request for a longer one... Until I read the original in the repo 😁

Reply:
;) - I'll add a bit to it as I add features!